compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -417,14 +417,14 @@ def cmd_start_endpoint(
         uname = pw_rec.pw_name
 
         if not self._allow_same_user:
-            p_uname = self._mt_user.pw_name
+            p_uname = self._mu_user.pw_name
             if uname == p_uname or uid == os.getuid():
                 raise InvalidUserError(
-                    "Requested UID is same as multi-tenant UID, but configuration"
-                    " has not been marked to allow the multi-tenant user to process"
+                    "Requested UID is same as multi-user UID, but configuration"
+                    " has not been marked to allow the multi-user user to process"
                     " tasks.  To allow the same UID to also run user endpoints,"
                     " consider using a non-root user or removing privileges from UID."
```

Comment:
Similar comment as above.  "... allow the multi-user user to process ..." is awkward.  I'll leave it to you finesse, or can offer an edit if that would be helpful.

(And suggest a qualifier of "single-" in the second sentence as well.)

Reply:
>can offer an edit if that would be helpful

Please do! Haven't come up with anything particularly great myself.

Reply:
```diff
- "Requested UID is same as multi-tenant UID, but configuration"
- " has not been marked to allow the multi-tenant user to process"
- " tasks.  To allow the same UID to also run user endpoints,"
- " consider using a non-root user or removing privileges from UID."
+ "Requested UID is same as multi-user UID, but configuration"
+ " has not been marked to allow the multi-user UID to process"
+ " tasks.  To allow the multi-user UID to also run single-user"
+ " endpoints, consider using a non-root user or removing privileges"
+ " from the UID."

```

For general edification, note that I _intentionally_ did not mention the flag that would override this in the configuration.  I want user-admins to go digging for that if they truly, truly need it.  [For reference.](https://github.com/funcx-faas/funcX/blob/main/compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py#L135-L146 "Allow same-user override warning") [`endpoint_manager.py`]